src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+function getEmployees() {
+  const employeeList = [];
+  const employeeListElements = document.querySelectorAll('li');
+
+  for (const employeeElement of employeeListElements) {
+    const employee = {};
+
+    employee.name = employeeElement.innerText;
+    employee.position = employeeElement.dataset.position;
+
+    employee.salary = +employeeElement.dataset.salary
+      .replaceAll(',', '')
+      .replace('$', '');
```

Comment:
The salary parsing logic is correct, but it could be improved for clarity. Consider using a single line with a regular expression to remove non-numeric characters, which might be more efficient and easier to read.